Reference/V9-Config/GlobalSettings/index.md

```diff
@@ -33,6 +33,7 @@ To make it easier to see what values are present in the global section, the foll
       "DisableElectionForSingleServer": false,
       "DatabaseFactoryServerVersion": "SqlServer.V2019",
       "MainDomLock": "MainDomSemaphoreLock",
```

Comment:
Is this correct? Below `IMainDomLock` is mentioned.

Reply:
Yep, the configuration value `MainDomLock` controls the implementation of `IMainDomLock`